src/cyc_limits.h

```diff
@@ -37,14 +37,14 @@ static const double float_ulp_eq = 2;
 /// maximum value for a function modifier (i.e., a_i for variable x_i)
 static const double kModifierLimit = pow(10, 10);
 
-/// maximum (+ or -) value for an integer variable
-static const int kIntBoundLimit = std::numeric_limits<int>::max();
+/// maximum (+) value for an integer variable
+static const int CY_LARGE_INT = std::numeric_limits<int>::max();
```

Comment:
Sure would be nice to define these all in one unique location... but I suppose that may be pretty challenging.

Reply:
See the latest [commit](https://github.com/cyclus/cyclus/pull/1757/commits/63d4c4f0042d8b92f5f62c964f3948572d9687e1), is this what you had in mind?

Reply:
Spoke too soon. CI failing now.